File: Day 10/homework/homework.py
#1)ამ ფოტოს მიხედვით გააკეთეთ პროგრამა. (მოხმარებელს შემოატანინეთ პაროლი იქამდე სანამ არ დაემტხვევა რეგისტრირებულ პაროლს, 
#while ციკლის და  if else _ის გამოყენებით)


authorized = False
password = "saba123"

while authorized != True:
    user_input = input("Please enter your password: ")
    if user_input == password:
        print("Access Granted")
        authorized = True

File: Day 163/homework/homework.js
import { useState } from "react";

const Component2 = () => {
  const [inputedInfo, setInputedInfo] = useState(() => {
    const savedInfo = localStorage.getItem("userInfo");
    return savedInfo ? JSON.parse(savedInfo) : null;
  });
  
  const styleDiv = {
    display: "flex",
    flexDirection: "column",
    alignItems: "center",
    justifyContent: "center",
    gap: 10,
  }
  
  const handleSubmit = (e) => {
    e.preventDefault();
    console.log('Submitted');

    const info = {
      fullname: e.target.fullname.value,
      email: e.target.email.value,
      password: e.target.Pass.value,
    }

    setInputedInfo(info);
    localStorage.getItem("userInfo", JSON.stringify(info));
  }
  
  return (
    <main>
      <div style={{ ...styleDiv }}>
        <h1>Sign up</h1>
        <form style={{ ...styleDiv }} onSubmit={handleSubmit}>
          <input type="text" name="fullname" placeholder="fullname" />
          <input type="email" name="email" placeholder="Email" />
          <input type="password" name="Pass" placeholder="Password" />

          <button>Submit</button>

          <div>
            {inputedInfo ? (
              <div>
                <h2>Fullname: {inputedInfo.fullname}</h2>
                <h2>Email: {inputedInfo.email}</h2>
                <h2>Password: {inputedInfo.password}</h2>
              </div>
            ) : (
              <h2>Please enter info!</h2>
            )
            }
          </div>
        </form>
      </div>
    </main>
  )
}

export default Component2;